go.mod

```diff
@@ -2,7 +2,7 @@ module github.com/seanavery/viam-droid-cam
 
 go 1.21.5
 
-require go.viam.com/rdk v0.19.1
+require go.viam.com/rdk v0.20.1-0.20240209172210-8dc034cf4d2a
```

Comment:
[note] `v0.20.0` did not include CGO changes. So using more recent commit. Builds working on macos.